pyglmnet/__init__.py

```diff
@@ -1,4 +1,4 @@
 from .pyglmnet import GLM, set_log_level
 from .utils import softmax, label_binarizer, log_likelihood
 from .datasets import fetch_tikhonov_data
-__version__ = '1.0.0'
+__version__ = '1.0.1'
```

Comment:
normally we do `0.1` then `0.2`, upto `0.10` ... you keep going till you feel confident that you can do a `1.0` release. And then for each of these versions, say `0.7`, you can have sometimes a maintenance version like `0.7.1` or `0.7.2` if you screwed up something during your `0.7` release. But normally not more than one or two maintenance versions. And during development, you append `dev0` to the version. So, something like `0.8.dev0` if you're developing for the `0.8` version. See here for the guidelines: https://www.python.org/dev/peps/pep-0440/